Cpix/ContentKey.cs

```diff
@@ -73,7 +73,7 @@ private void ValidateEntity(CpixDocument document)
 
 		private void ValidateContentKeyValueAndSize(CpixDocument document)
 		{
-			if (Value == null || !Constants.ValidContentKeyLengthsInBytes.Contains(Value.Length))
+			if (Value != null && !Constants.ValidContentKeyLengthsInBytes.Contains(Value.Length))
```

Comment:
Perhaps leave a comment here:
"We support content keys without a value because many packagers create such documents for requesting keys from key services, which then fill in the value."